### ISSUE TITLE
The "End Turn" button doesn't get removed when the game is over.

### ISSUE THREAD

**Issue by misbach** [parke/dicewars#1]
After the game ends, the "End Turn" button still remains.

**Comment by parke** [comment]
> After the game ends, the "End Turn" button still remains.

Thanks for the bug report!  I've actually been aware of that issue for two years.  But given that the workaround is simply to reload the webpage, I haven't been motivated to fix it.

I actually already have the fix (sort of) because I actually modified Dicewars twice, on two different computers.  On the first computer, the "End Turn" button disappears (as it should).  On the second computer (the computer I used for the published copy) the button remains.  So by diff'ing the two codebases, I should be able to see what I did differently on the first computer.

Aside:  I lack a clear understanding of the structure of the Dicewars source code.  All my changes were fairly crude.  I'd make some changes and then run Dicewars to see if I had achieved my intended result.  So the process of making changes was sort of like hacking through a jungle with a machete.

What do you think of Dicewars itself as a game?

**Comment by misbach** [comment]
I've been addicted to dicewars since it came out over a decade ago lol

**Comment by parke** [comment]
> I've been addicted to dicewars since it came out over a decade ago lol

Over two decades ago, I believe.  The copyright date is 2001.

And you've been watching the (time consuming) dice roll animations the whole time?

**Comment by misbach** [comment]
two decades 😱!
Yes, I have. Thanks for the fast version :-)

**Comment by parke** [comment]
Change d4eb144 should hide the `End Turn` button at the end of the game.

The bug was tricky to find because the bug was not in the end game logic.  The bug was in the battle logic.

The bug was caused by an extra line that I mistakenly commented out in my second manually edited `main.js` file.  And this extra commented line did not show up in the `diff` output when I diffed GameDesign's original `main.js` against either of my two edited `main.js` files.  And because of noise in my two edited versions of `main.js`, I never diffed them against each other.
